first-react-app/src/api/lastFMAPI.ts

```diff
@@ -0,0 +1,114 @@
+import fetch from "cross-fetch";
+
+interface getArtistParams {
+  artist: string;
+  method: string;
+  limit: number;
+}
+
+interface getArtistParamsInfo {
+  artist: string;
+  method: string;
+}
+
+type queryParams = {
+  artist: string;
+  method: string;
+  limit?: number;
+  api_key: string;
+  format: string;
+};
+
+type queryKeys = keyof queryParams;
+
+export const lastFM = {
+  _apiLink: "http://ws.audioscrobbler.com/2.0/",
+  _apiKey: "d15ae01ae3b43dd6f2f28360c896b596",
+  _apiFormat: "json",
+
+  async getInfo(params: getArtistParams | getArtistParamsInfo) {
```

Comment:
you should use arrow function instead of regular function
```suggestion
  getInfo: async (params: getArtistParams | getArtistParamsInfo) => {
```